web/src/features/Dashboard/components/DashboardGraphsCard.tsx

```diff
@@ -321,16 +278,6 @@ export default class DashboardGraphsCard extends React.Component<Props, State> {
     );
   };
 
-  componentDidMount() {
-    if (this.props.prometheusAddress) {
-      this.state.getAppDashboardJob.start(this.getAppDashboard, 2000);
-    }
-  }
-
-  componentWillUnmount() {
-    this.state.getAppDashboardJob.stop();
-  }
-
```

Comment:
as far as i can tell, this component doesn't use its dashboard state for anything. It renders the data passed in from its parent. 

I did take this concept and used it to add a `refetchInterval` to the dashboard call in `Dashboard.tsx`- so now the Dashboard refetches `/dashboard` on an interval and passes that data down to the `DashboardGraphsCard`